contracts/UniswapV3Staker.sol

```diff
@@ -205,11 +205,9 @@ contract UniswapV3Staker is IUniswapV3Staker, IERC721Receiver, ReentrancyGuard {
         require(deposit.numberOfStakes == 0, 'NUMBER_OF_STAKES_NOT_ZERO');
         require(deposit.owner == msg.sender, 'NOT_YOUR_NFT');
 
-        // TODO: do we have to check for a failure here? Also double-check
```

Comment:
openzeppelin says use of `transferFrom` is discouraged in favor or `safeTransferFrom` a little extra protection against malicious `to` recipients https://docs.openzeppelin.com/contracts/2.x/api/token/erc721